projects/v2/src/app/components/home/home-routing.module.ts

```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+import { HomeComponent } from './home.component';
+
+const routes: Routes = [
```

Comment:
What does this file do? I don't think this is required. The `app-routing.module.ts` handles the routing.

Reply:
Yup, I removed that file.